From fc3ba6d701d31c688ed7955b8cfa8b223c1702e3 Mon Sep 17 00:00:00 2001
From: Lei Deng <denglei@eswincomputing.com>
Date: Thu, 20 Jun 2024 08:56:15 +0800
Subject: [PATCH 053/128] fix:Solving HDMI driver compilation issue.

Signed-off-by: Lei Deng <denglei@eswincomputing.com>
---
 drivers/gpu/drm/eswin/Kconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/eswin/Kconfig b/drivers/gpu/drm/eswin/Kconfig
index de77dc0d9101..77733cd77352 100644
--- a/drivers/gpu/drm/eswin/Kconfig
+++ b/drivers/gpu/drm/eswin/Kconfig
@@ -32,7 +32,9 @@ config ESWIN_DW_HDMI
     bool "ESWIN specific extensions for Synopsys DW HDMI"
     depends on DRM_ESWIN
     select CEC_CORE if CEC_NOTIFIER
-    select DW_HDMI
+    select DRM_DISPLAY_HDMI_HELPER
+    select DRM_DISPLAY_HELPER
+    select DRM_KMS_HELPER
     help
       This selects support for ESWIN SoC specific extensions
       for the Synopsys DesignWare HDMI driver. If you want to
-- 
2.47.0

