From 8f6b201a7ee5470a575085895a16f91a8bc29c0f Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Thu, 24 Oct 2024 08:06:07 +0000
Subject: [PATCH 105/128] UBUNTU: linux-premier-6.6.21-8.1

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 debian/changelog | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index a34a840970ed..96658566d266 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+linux-premier (6.6.21-8.1) noble; urgency=medium
+
+  * Disabled kernel debug configs 
+
+ -- Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>  Thu, 24 Oct 2024 08:04:20 +0000
+
 linux-premier (6.6.21-7.1) noble; urgency=medium
 
   * Optimized configs, fixed usb boot issue and bootlog errors
-- 
2.47.0

