From 34562d42c406e53f6a39a71f50f371d5c4a19e5e Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Fri, 20 Dec 2024 12:47:32 +0000
Subject: [PATCH 126/128] UBUNTU: linux-premier-6.6.21-10.6

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 debian/changelog | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 9a41bfc6948a..3a7df65e0a56 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+linux-premier (6.6.21-10.6) noble; urgency=medium
+
+  * Updated dependency for eswin-eic7x-gpu
+
+ -- Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>  Fri, 20 Dec 2024 12:46:25 +0000
+
 linux-premier (6.6.21-10.5) noble; urgency=medium
 
   * Removed Breaks for mesa-opencl-icd
-- 
2.47.0

